src/index.ts

```diff
@@ -4,8 +4,8 @@ import { connect, keyStores } from "near-api-js";
 import { updateState } from "./state/State";
 import Options from "./types/Options";
 
-export default async function init(options?: Options) {
-  const nearConfig = getConfig(process.env.NEAR_ENV || "testnet");
+export default async function init(options: Options) {
+  const nearConfig = getConfig(options.networkId);
```

Comment:
My next task will be to remove this async init entirely. To avoid fixing it in this PR, I've duplicated the `getConfig` call in here and `NearWalletSelector`.